data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/actions/CountAggregateAction.java

```diff
@@ -56,13 +61,25 @@ private long getTimeNanos(Instant time) {
         return currentTimeNanos;
     }
 
+    public Exemplar createExemplar(final Event event) {
+        long curTimeNanos = getTimeNanos(Instant.now());
+        Map<String, Object> attributes = event.toMap();
+        return new DefaultExemplar(
+                    OTelProtoCodec.convertUnixNanosToISO8601(curTimeNanos),
+                    1.0,
+                    event.get("spanId", String.class), // maybe null
```

Comment:
This value should only be found when this is a `SpanEvent`. It may be better to check if this is a span and then handle as necessary.

e.g.

```
if(event instanceof SpanEvent) {
  SpanEvent spanEvent = (SpanEvent) event;
  String spanId = spanEvent.getSpanId();
  String traceId = spanEvent.getTraceId();
}

return new DefaultExemplar(
  OTelProtoCodec.convertUnixNanosToISO8601(curTimeNanos),
  1.0,
  spanId,
  traceId,
  attributes);
```